lib/app/file.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-var ArrayStore = require('fh-wfm-mediator/lib/array-store');
-
-var files = [];
+var SimpleStore = require('fh-wfm-simple-store');
 
 module.exports = function(mediator) {
-  var arrayStore = new ArrayStore('file', files);
-  arrayStore.listen('cloud:', mediator);
+  var fileStore = new SimpleStore('file');
+
+  fileStore.list().then(function(res) {
+    if (!res || res.length === 0) {
+      fileStore.init();
+    }
+
+    fileStore.listen(':cloud', mediator);
```

Comment:
the `:cloud` is repeated several times. It would be nice to create a constant / config value for it.